packages/web/src/components/artist/ArtistPopover.tsx

```diff
@@ -74,7 +75,12 @@ export const ArtistPopover = ({
 
   const content =
     creator && userId !== creator.user_id ? (
-      <ArtistCard artist={creator} />
+      <ArtistCard
+        artist={creator}
+        onNavigateAway={() => {
```

Comment:
Would we just want this to be onClick possibly? Like I assume whenever the user actually clicks on the popever, we want to close it?

Reply:
Hmm interesting product implications of that - if they just click on the cover photo or some other non-actionable spot, I assumed we wouldn't want to dismiss? It would for sure clean this up a ton though